src/index.html

```diff
@@ -15,8 +15,98 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="image">
+        <a href="index.html">
+          <img
+            src="images/logo.png"
+            alt="logo_moyo"
+          />
+        </a>
+      </div>
+
+      <nav class="nav">
```

Comment:
Add an empty line between the `header` and `nav` elements to improve readability, as they are sibling elements and the `nav` is a multiline block.